charts/rancher-vsphere-csi/values.yaml

```diff
@@ -166,6 +169,9 @@ versionOverrides:
           csiProvisioner:
             repository: rancher/mirrored-sig-storage-csi-provisioner
             tag: v3.5.0
+          csiSnapshotter:
+            repository: rancher/mirrored-sig-storage-csi-snapshotter
+            tag: v6.2.2
```

Comment:
v6.2.2 is used in the upstream `vsphere-csi-driver` release v3.1.2
https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/v3.1.2/manifests/vanilla/vsphere-csi-driver.yaml#L404